GeoRef/georefimage.cs

```diff
@@ -247,7 +247,7 @@ private DateTime getPhotoTime(string fn)
                             int latindex = dflog.FindMessageOffset("GPS", "Lat");
                             int lngindex = dflog.FindMessageOffset("GPS", "Lng");
                             int altindex = dflog.FindMessageOffset("GPS", "Alt");
-                            int raltindex = dflog.FindMessageOffset("GPS", "RAlt");
+                            int raltindex = dflog.FindMessageOffset("GPS", "RelAlt");

```

Comment:
where did this change come from?
https://github.com/diydrones/ardupilot/blob/master/libraries/DataFlash/DataFlash.h#L723